.gitignore

```diff
@@ -59,3 +59,5 @@ typings/
 
 # next.js build output
 .next
+
+secretKey.js
```

Comment:
Consider creating example files with empty/dummy values for gitignored config files. For example, secretKey.js.example and variables.env.example.